### PR TITLE
103 fix accessibility across the site

### PR DESCRIPTION
**Pull request by megansizemore**
Still need to:
-go over color contrast on a few pages
-add fixes for contact, weekofcode, india pages, header/banner, and intro-to-web-development pages

### CODE REVIEW

**Comment by megansizemore** [comment]
zoho forms aren't fully accessible but there isn't anything that we can do. Color contrast issues should be fixable if someone can customize the forms through zoho though

**Comment by raney24** [comment]
@megansizemore I'm going to leave a few comments throughout this PR, let me know if you have any questions about them

File: web/src/pages/calendar/index.js [hunk 1]
```diff
@@ -9,16 +9,18 @@ import SEO from '../../components/seo';
 const Calendar = () => {
   return (
     <Layout>
+      <main>
```
Comment (raney24) [inline web/src/pages/calendar/index.js:12]: @megansizemore would it be okay to move the `<main>` tags to the `<Layout>` file? This way, we don't have to remember to do it for each file.

**Review by raney24 — COMMENTED** [review]
I made a few comments, let me know if you have any questions. 

Great job on this, it was a big PR!

File: web/src/pages/calendar/index.js (hunk 1 above)
Comment (megansizemore) [reply web/src/pages/calendar/index.js]: @raney24 yes, I just made those changes. thanks for catching that!

**Pull request merged**